### PR TITLE
feat(project): once pu are set, trigger relevant event

### PR DESCRIPTION
**Pull request by kgajowy**
https://vizzuality.atlassian.net/browse/MARXAN-995

### CODE REVIEW

File: api/apps/api/src/modules/planning-units/planning-units.service.ts [hunk 1]
```diff
@@ -1,79 +1,32 @@
...
-    const { projectId } = job.data;
-    await this.events.createIfNotExists({
-      kind: API_EVENT_KINDS.project__planningUnits__finished__v1__alpha,
-      topic: projectId,
-      externalId: eventId,
-    });
-  }
+    // bad typing - may happen that job wasn't added
+    if (!job) {
+      return;
```
Comment (ikas) [inline api/apps/api/src/modules/planning-units/planning-units.service.ts:24]: I'd suggest adding some logging here - if adding jobs starts to fail for some reason, we'd want to easily be able to track where that happens

**Review by ikas — APPROVED** [review]
Minor comment, overall LGTM :+1:

File: api/apps/api/src/modules/planning-units/planning-units.service.ts (hunk 1 above)
Comment (kgajowy) [reply api/apps/api/src/modules/planning-units/planning-units.service.ts:29]: @ikas Added in https://github.com/Vizzuality/marxan-cloud/pull/643/commits/bedc454297d448da6ba67299a25837e7b9cd1058

Comment (ikas) [reply api/apps/api/src/modules/planning-units/planning-units.service.ts:29]: nice :ok_hand:

**Pull request merged**